rllib/algorithms/impala/impala.py

```diff
@@ -665,22 +649,18 @@ def aggregate_into_larger_batch():
 
     def get_samples_from_workers(
         self,
-    ) -> Dict[
-        Union[ActorHandle, RolloutWorker], List[Union[ObjectRef, SampleBatchType]]
-    ]:
+        return_objects: Optional[bool] = False,
```

Comment:
Can we call this `return_object_refs`? Otherwise, this would be a very confusing name, I think.

Reply:
done. good catch.